src/ws-header/ws-header.js

```diff
@@ -156,14 +165,14 @@ export class WSHeader extends Component {
       this.dispatchEvent('ws-auth-changed', accessToken);
     });
     // Check if we was redirected from the auth page and an access token is available
-    this.authorization.tryFetchToken(location.hash.substr(1));
+    this.constructor.authorization.tryFetchToken(location.hash.substr(1));
     // Listen for authentication requests
     window.addEventListener('ws-authorize', () => {
-      this.authorization.authorize();
+      this.constructor.authorization.authorize(props.loginUrl, props.clientId, props.businessPartnerId);
```

Comment:
Where does this businessPartnerId comes from ? Are we allowed to share this here publicly ? I don't want to ran into any sort of security / compliance issue as we expose our IAM endpoint + some id

Reply:
Actually I don't use PropTypes and just mock the functions. So PropTypes.string.isRequired would lead to an undefined error for me.

Reply:
ok, are you handling such error ? E.g. in authorization.js -> buildQuery throwing some Error ? 😄  or `console.error` so people get immediate feedback ? I know it is not necessary as rendering and using it will throw an error anyway